cmd/task/task.go

```diff
@@ -206,6 +208,10 @@ func main() {
 		e.InterceptInterruptSignals()
 	}
 
+	if e.Taskfile.Interval == "" {
+		e.Taskfile.Interval = strings.TrimSpace(interval)
+	}
```

Comment:
For consistency with other configurations, we should have an attribute on the `Executor` struct instead of setting in the Taskfile here. See `Silent` for example.

This allow more flexibility for those who use Task as a library.

The priority order is: CLI flag > Taskfile > Default (5s).